.travis.yml

```diff
@@ -0,0 +1,87 @@
+language: c
+branches:
+  only:
+  - master
+
+dist: xenial
+
+cache:
+  directories:
+    - $HOME/.cabsnap
+
+matrix:
+  include:
+    - env: TEST=MAIN GHC_VER=8.4.4 BUILD=CABAL CABAL_VER=2.2 STDLIB_VER=1.2
```

Comment:
Why this particular version of ghc and cabal?

Reply:
You have to pick a version, don't you? This one works so far (it's what we
are using for the stdlib).

Reply:
Definitely, a version needs to be picked. I was sort-of hoping to find a way to pick it consistently with the choice from stdlib, in a way that would not require constant maintenance.

Maybe a dummy 'build' project (aka under agda, i.e. `agda/build` parallel to `agda/agda`, etc) that would contain these choices in a central way. For example.

Reply:
Ideally I would want nightly builds of Agda so that all software projects
could just pull whatever is the latest without having to pin down a commit
known to build (although Agda has gotten way better at not breaking master)
and re-build from scratch.

No time to look into this though.